apps/package.json

```diff
@@ -75,18 +73,13 @@
     "phantomjs": "wjordan/phantomjs#v2.0.0",
     "react": "0.13.3",
     "react-tools": "0.13.3",
```

Comment:
This is one of the ones that I removed initially, but then saw failures. So it seems like the answer is yes